src/views/HomeView.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+    <content-layout>
+        <button @click="getPreviewCourses">get token</button>
+    </content-layout>
+</template>
+
+<script setup lang="ts">
+import ContentLayout from "@/layouts/ContentLayout.vue";
+import api from "@/api";
+
+const getPreviewCourses = () => {
+    api.get('/core/preview-courses/').then((res) => {
+        console.log(res);
+    });
+}
```

Comment:
I don't know the context, but it seems that here's no error tracking. Is there any sense to add `catch(e)` here&